src/components/PhoneNumberInput/index.js

```diff
@@ -16,19 +21,31 @@ const FlagComponent = ({ countryCode, flagsPath }) => (
   />
 );
 
-const PhoneNumberInput = ({ translate, clearErrors, actions = {}}) => {
+const PhoneNumberInput = ({ translate, clearErrors, actions = {}, smsNumberCountryCode}) => {
 
   const onChange = (number) => {
     clearErrors()
     const valid = isValidPhoneNumber(number)
     actions.setMobileNumber({number, valid})
   }
 
+  const validCountryCode = () => {
```

Comment:
I think validation should be done at init time, not at flow time.

Reply:
is `react-phone-number-input/modules/countries` big? If not it's not necessary to load that lazily in the root file.

Reply:
I have refactored it to do the validation at init time. `react-phone-number-input/modules/countries` is not a large file, it only has an array of about 240 elements.